serving/src/main/java/ai/djl/serving/ModelServer.java

```diff
@@ -449,4 +462,28 @@ private static void printHelp(String msg, Options options) {
         formatter.setWidth(120);
         formatter.printHelp(msg, options);
     }
+
+    private String createHuggingFaceModel(String modelId) throws IOException {
+        String hash = Utils.hash(modelId);
+        String downloadDir = Utils.getenv("SERVING_DOWNLOAD_DIR", null);
+        Path parent = downloadDir == null ? Utils.getCacheDir() : Paths.get(downloadDir);
+        parent = parent.resolve(modelId);
+        Path huggingFaceModelDir = parent.resolve(hash);
+        if (!Files.exists(huggingFaceModelDir)) {
```

Comment:
Should we return null if File already exists?
If the model already registered in config.properties, this may lead to conflict